index.md

```diff
@@ -15,6 +21,10 @@ The goal with this guide is to give contributors confidence when making a contri
 
 This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct), and this extends to the contribution process. For more information, please see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq).
 
+## Contribution process
+
+The AWS CLI has defined a process to guide contributions from opening a new issue to getting a contribution merged. You can read more about that on our {doc}`contribution process page <contribution_process>`. 
```

Comment:
Maybe phrase this to `The AWS CLI project has a process...` specifically removing defined. It read awkward with it. 

It also may be worth adding a sentence on the intended audience for the contribution process page.